lib/Onyx.js

```diff
@@ -783,6 +783,7 @@ function init({
 const Onyx = {
     connect,
     disconnect,
+    getAllKeys,
```

Comment:
I think we can just see which keys exist by subscribing to the collection? If the new value that comes back from the API is different then what we have [here](https://github.com/Expensify/App/blob/21b1ee466b7084b3d6d11602d6b09e368ab02f88/src/libs/actions/Policy.js#L14) then just remove those keys?

Reply:
Ah, very true. Closing this PR, since we no longer need it. Thanks for all the input!